cli/src/complete.rs

```diff
@@ -1058,8 +1058,7 @@ fn get_jj_command() -> Result<(JjBuilder, UserSettings), CommandError> {
     let cwd = std::env::current_dir()
         .and_then(dunce::canonicalize)
         .map_err(user_error)?;
-    // No config migration for completion. Simply ignore deprecated variables.
```

Comment:
nit: this comment is about `jj_lib::config::migrate()`, not about macOS legacy config dir.